src/plugins/intel_gpu/src/graph/primitive_inst.cpp

```diff
@@ -429,6 +433,47 @@ void primitive_inst::build_deps() {
     }
 }
 
+void primitive_inst::rebuild_deps(
+    std::unordered_map<primitive_id, std::shared_ptr<primitive_inst>> const& primitives) {
+
+    _deps.resize(_dep_ids.size());
+    for (size_t i = 0; i < _dep_ids.size(); i++) {
+        if (primitives.count(_dep_ids[i]) > 0) {
+            _deps[i] = primitives.at(_dep_ids[i]);
+        } else {
+            std::cout << _dep_ids[i] << " is not found in _primitives" << std::endl;
```

Comment:
Add assert instead of print?

Reply:
It's replaced with `OPENVINO_ASSERT`. Thank you.